app/css/components/tooltips/concept.css

```diff
@@ -91,4 +93,27 @@
             @apply text-darkGray leading-paragraph font-medium;
         }
     }
+
+    &::before {
+        content: "";
+        width: 100%;
+        height: 8px;
+        left: 0;
+        top: 0;
+        display: inline-block;
```

Comment:
Is this needed?

Reply:
Yes so that it can be used to cover the gap between the <concept> and <concepttooltip>

Reply:
I think `display: block` might work too, but untested

Reply:
Yeah. I think you can remove this line altogether and it will default to `display: block`.